docs/core/howto/defer-intro.rst

```diff
@@ -382,6 +383,11 @@ Inline callbacks - using 'yield'
     Unless your code supports Python 2 (and therefore needs compatibility with older versions of Twisted), writing coroutines with the functionality described in "Coroutines with async/await" is preferred over ``inlineCallbacks``.
     Coroutines are supported by dedicated Python syntax, are compatible with ``asyncio``, and provide higher performance.
 
+    Since Twisted NEXT existing ``inlineCallbacks``-based code can be converted to coroutines function-by-function.
```

Comment:
Minor comment.

Maybe use the dedicated verion marker.

```suggestion
.. versionadded:: Twisted NEXT

    Existing ``inlineCallbacks``-based code can be converted to coroutines function-by-function.
```

Reply:
Looks better, thanks.